content/en/blog/_posts/2023-03-22-recent-developments-in-kubelet-to-speed-up-pod-startup.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: "Recent developments in kubelet to speed up Pod startup"
+date: 2023-03-21T16:00:00+0000
```

Comment:
Is this a post-release blog article? “Recent” suggests these have already shipped, but v1.27 is not released.

Reply:
Some features mentioned here will be introduced after v1.27. 
I should change this PR to target dev-1.27. Should I change the date to v1.27 release date or keep it as?

Reply:
It seems that I should update the date here once all content got approved.